src/pykeen/models/predict.py

```diff
@@ -1,25 +1,140 @@
 # -*- coding: utf-8 -*-
 
-"""Prediction workflows."""
+"""
+Prediction workflows.
```

Comment:
This may have overlap with the (potentially now outdated) predictions tutorial from the docs

Reply:
this is pretty important to resolve before merging this. Ideally, we can continue to provide high-level wrappers that do the same thing, but if that is a huge pain to maintain then maybe not

Reply:
https://github.com/pykeen/pykeen/pull/1090/commits/e8a4f4af96db5db32b7517d70490b3a013521a5a